weblate/accounts/admin.py

```diff
@@ -92,6 +94,29 @@ def user_groups(self, obj):
         """
         return ','.join([g.name for g in obj.groups.all()])
 
+    def __init__(self, *args, **kwargs):
+        super(WeblateUserAdmin, self).__init__(*args, **kwargs)
+
+        groups = Group.objects.all()
```

Comment:
Apparently you can not access database in admin __init__ as this is called on startup even before database migrations are executed, so it has to work completely without database. With your changes it's impossible to run initial migration (see errors in tests).